<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlumnoWindow</class>
 <widget class="QMainWindow" name="AlumnoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alumno</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="etiqueta_correo_alimno">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Correo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="etiqueta_semestre_alumno">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Semestre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="etiqueta_telefono_alumnos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tel√©fono:</string>
    </property>
   </widget>
   <widget class="QLabel" name="etiqueta_edad_alumnos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="etiqueta_nombre_alumnos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="etiqueta_fecha_alumnos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fecha ingreso:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_correo_alumnos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_semestre_alumnos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_telefono_alumnos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_edad_alumnos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_nombre_alumnos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_fecha_alumnos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_agregar_alumno">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_eliminar_alumno">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mensaje_alumno">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mensaje</string>
    </property>
   </widget>
   <widget class="QLabel" name="etiqueta_cursa_alumnos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cursa:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_cursa_alumnos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
